molgenis-one-click-importer/src/test/java/org.molgenis.oneclickimporter/controller/OneClickImporterControllerTest.java

```diff
@@ -106,17 +103,16 @@ public void testXLSXFileImport() throws Exception
 		Sheet sheet = mock(Sheet.class);
 		DataCollection dataCollection = mock(DataCollection.class);
 		EntityType table = mock(EntityType.class);
-		String tableId = "genrated_table_id";
+		String tableId = "generated_table_id";
 		File file = mock(File.class);
 		when(fileStore.store(any(InputStream.class), anyString())).thenReturn(file);
 		when(excelService.buildExcelSheetFromFile(file)).thenReturn(sheet);
 		when(oneClickImporterService.buildDataCollection("simple-valid", sheet)).thenReturn(dataCollection);
-		when(entityService.createEntity(dataCollection)).thenReturn(table);
+		when(entityService.createEntityType(dataCollection)).thenReturn(tableId);
 		when(table.getId()).thenReturn(tableId);
 
 		mockMvc.perform(fileUpload(OneClickImporterController.URI + "/upload").file(multipartFile))
-				.andExpect(status().isCreated())
-				.andExpect(header().string("Location", endsWith(tableId)));
+			   .andExpect(status().isOk());
```

Comment:
it is creating a entity and should really return created, and setting the Location is a nice way to make a usable web api